setup.py

```diff
@@ -46,6 +46,7 @@ def find_version(*paths):
         "typing-extensions == 4.4.0",
         "pymemcache == 4.0.0",
         "redis == 4.6.0",
+        "schedule == 1.2.0",
```

Comment:
Do we need to add this to `requirements.txt`/`dev-requirements.txt` as well?

Reply:
It's already added. We didn't add it to setup.py earlier.